ppx/ppx.ml

```diff
@@ -1215,48 +1215,22 @@ let jsxMapper () =
             let empty_loc = Location.in_file filename in
             let binding_pat_loc = empty_loc in
             let outer_make expression =
-              let react_component_attr =
-                try Some (List.find hasAttr pval_attributes)
-                with Not_found -> None
-              in
-              let payload =
-                match react_component_attr with
-                | Some {attr_payload} ->
-                    Some attr_payload
-                | None ->
-                    None
-              in
-              let make_js_comp ~loc ~fn_name ~named_type_list rest =
-                let props = get_props_attr payload in
+              let make_js_comp ~loc ~fn_name rest =
```

Comment:
I believe we can remove `make_js_comp` altogether to make the code simpler, as it brings no value anymore.